psychopy/data.py

```diff
@@ -3464,8 +3464,8 @@ class QuestHandler(StairHandler):
         # create staircase object
         # trying to find out the point where subject's response is 50 / 50
         # if wanted to do a 2AFC then the defaults for pThreshold and gamma
-        # are good
-        staircase = data.QuestHandler(staircase._nextIntensity, 0.2,
+        # are good. As start value, we'll use 50% contrast, with SD = 20%
+        staircase = data.QuestHandler(0.5, 0.2,
```

Comment:
Also, I am pretty certain `startVal` does not necessarily correspond to the first proposed intensity, i.e., stating that `As start value, we'll use 50% contrast` (with `startVal=0.5`) could be misleading. `startVal` specifies our _prior assumption about threshold location_ (together with `startValSd`), but this does not mean that QUEST will actually "start" there, does it?